sb3_contrib/common/utils.py

```diff
@@ -67,3 +68,82 @@ def quantile_huber_loss(
     else:
         loss = loss.mean()
     return loss
+
+
+# TODO: write regression tests
+def cg_solver(Avp_fun: Callable[[th.Tensor], th.Tensor], b, max_iter=10, residual_tol=1e-10) -> th.Tensor:
```

Comment:
as a general remark, please use meaningful names and avoid mathematical notation when possible.
For instance here `cg_solver` -> `conjugate_gradient_solver`
`Avp_fun` -> ?